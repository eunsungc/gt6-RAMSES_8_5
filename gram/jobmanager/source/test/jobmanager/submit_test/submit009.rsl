&(executable=/bin/sh)
 (arguments = -c "grid-proxy-info -type")
 (library_path = $(GLOBUS_LOCATION)/lib)
