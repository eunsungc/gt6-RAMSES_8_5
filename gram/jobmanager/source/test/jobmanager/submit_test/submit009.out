RFC 3820 compliant limited proxy
